end_to_end_tests/metadata_snapshots/uv.pyproject.toml

```diff
@@ -11,23 +11,16 @@ dependencies = [
     "python-dateutil>=2.8.0,<3",
 ]
 
-[tool.hatch.build.targets.sdist]
-include = [
-    "test_3_1_features_client",
+[tool.uv.build-backend]
+module-name = "test_3_1_features_client"
+module-root = ""
+data = [
     "CHANGELOG.md",
-    "test_3_1_features_client/py.typed",
-]
-
-[tool.hatch.build.targets.wheel]
-include = [
-    "test_3_1_features_client",
-    "CHANGELOG.md",
-    "test_3_1_features_client/py.typed",
 ]
 
 [build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
+requires = ["uv_build>=0.8.0,<1"]
```

Comment:
Is this what it populates by default? Normally I'd say `<0.9` but if `uv` puts `<1` I guess we should trust it

Reply:
It puts `<0.9` by default. It's also in their docs: https://docs.astral.sh/uv/concepts/build-backend/#using-the-uv-build-backend

However, the other backends `poetry / pdm` we don't specify a maximum. so I thought we might want a wider one here too.

Reply:
We should bound Poetry too IMO, maybe in a separate PR?

- https://python-poetry.org/docs/basic-usage/#project-setup

```toml
[build-system]
requires = ["poetry-core>=2.0.0,<3.0.0"]
build-backend = "poetry.core.masonry.api"
```

Reply:
Non-bounded build dependencies have been causing real issues (mostly in the Setuptools world) so I think it's best to generate bounded wherever the upstream tools recommend it.

Reply:
Good point @johnthagen . 🙇 I've pushed an update to this PR to pin `uv_build` to `>=0.8.0,<0.9.0`.

and I've opened a PR to pin `poetry` as well. https://github.com/openapi-generators/openapi-python-client/pull/1291

Thank you!